packages/scripts/scripts/format-js.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+const { sync: dirGlob } = require( 'dir-glob' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	fromConfigRoot,
+	fromProjectRoot,
+	getArgFromCLI,
+	getFileArgsFromCLI,
+	hasArgInCLI,
+	hasPackageProp,
+	hasProjectFile,
+} = require( '../utils' );
+
+// See: https://prettier.io/docs/en/configuration.html
+const hasProjectPrettierConfig =
+	hasProjectFile( '.prettierrc' ) ||
+	hasProjectFile( '.prettierrc.json' ) ||
+	hasProjectFile( '.prettierrc.yaml' ) ||
+	hasProjectFile( '.prettierrc.yml' ) ||
+	hasProjectFile( '.prettierrc.js' ) ||
+	hasProjectFile( '.prettierrc.config.js' ) ||
```

Comment:
```suggestion
	hasProjectFile( 'prettierrc.config.js' ) ||
```

No leading `.` period, Prettier uses cosmiconfig, see https://prettier.io/docs/en/configuration.html